### ISSUE TITLE
"Igel" Needs Improvements

### ISSUE THREAD

**Issue by Karkus476** [SuperTux/data#45]
This may need a corresponding issue in the SuperTux/supertux repo.
Why do these enemies take different amount of fireballs to kill?
The game should indicate to the user what effect fireballs are having.
Possible improvements may be needed to artwork too.
[This is the creature I'm talking about](https://github.com/SuperTux/data/blob/master/images/creatures/igel/walk-0.png)

**Comment by uafr** [comment]
It allways takes the same amount of one fireball, but you have to hit the vulnerable face.

**Comment by Rusty-Box** [comment]
Every time I attack Igel, I can only defeat it, when it reaches the edge of a platform or the ground.

**Comment by Karkus476** [comment]
@uafr Yes, that appears to be the case. This is not obvious to the player though, and some change needs to be made in my opinion, to make this clear. I've added needs-discussion.
